app/schema.graphql

```diff
@@ -795,6 +802,32 @@ type TimeSeriesDataPoint {
   value: Float
 }
 
+type Trace {
+  traceId: ID!
```

Comment:
Nice - I think I do somewhat wish we could make traces globally unique but I know we've moved away from that... We should think about this when we need to link back to traces from datasets.